x/ibc-address-limiter/README.md

```diff
@@ -0,0 +1,307 @@
+# IBC Rate Limit
```

Comment:
This README is about osmosis module. Remove all content and write a minimal description about our module.